### ISSUE TITLE
Crowbar Bug

### ISSUE THREAD

**Issue by Solanum-Imago** [Gardenstation/Gardenstation#41]
# Description

When unpowered, airlocks can be opened with crowbars and jaws of life, but cannot be closed with those same tools. You are able to interact with them, and a progress bar appears, and a sound effect plays upon completion, but the door's state remains unchanged.
